### ISSUE TITLE
[Bug] @farmfe/plugin-sass cannot find exe_path

### ISSUE THREAD

**Issue by niuiic** [farm-fe/farm#660]
### Steps to reproduce

[This line](https://github.com/farm-fe/farm/blob/9ba586f6b8e9b94c355c4c5d39096f760995d1bf/rust-plugins/sass/src/lib.rs#L226) mentions a path `dart-sass-embedded` which no longer exists in latest `sass-embedded-linux-x64`.

![image](https://github.com/farm-fe/farm/assets/63799565/2a659a9d-b6e3-4dbe-84a1-923b30e96da7)

I don't know how you want to fix it, so I don't open a pull request.

I think there is no need to try, but if you want, follow the steps.

```sh
git clone https://github.com/niuiic/gallery
pnpm i
pnpm build --filter css
```

### What is expected?

build successfully

### What is actually happening?

```
css:build: cache miss, executing 8ba6c5af54c37d1a
css:build:
css:build: > css@0.0.0 build /home/niuiic/Documents/projects/gallery/apps/css
css:build: > farm build
css:build:
css:build: [ Farm ] Using config file at /home/niuiic/Documents/projects/gallery/apps/css/farm.config.ts
css:build: thread '<unnamed>' panicked at '
css:build:  sass-embedded init error: No such file or directory (os error 2),
css:build:  Please try to install manually. eg:
css:build:  pnpm install sass-embedded-linux-x64
css:build: ', rust-plugins/sass/src/lib.rs:140:9
css:build: note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
css:build: fatal runtime error: Rust cannot catch foreign exceptions
css:build:  ELIFECYCLE  Command failed.
css:build: ERROR: command finished with error: command (/home/niuiic/Documents/projects/gallery/apps/css) pnpm run build exited (-1)
css#build: command (/home/niuiic/Documents/projects/gallery/apps/css) pnpm run build exited (-1)
```

### System Info

```Shell
System:
    OS: Linux 6.1 Gentoo Linux
    CPU: (16) x64 AMD Ryzen 7 5800H with Radeon Graphics
    Memory: 12.25 GB / 15.59 GB
    Container: Yes
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.6.1 - /run/user/1000/fnm_multishells/139909_1698305850673/bin/node
    npm: 9.8.1 - /run/user/1000/fnm_multishells/139909_1698305850673/bin/npm
    pnpm: 8.9.2 - /run/user/1000/fnm_multishells/139909_1698305850673/bin/pnpm
  npmPackages:
    @farmfe/core: ^0.13.12 => 0.13.12
```
```


### Any additional comments?

_No response_

**Comment by wre232114** [comment]
You can remove manually installed sass-embedded installed at https://github.com/niuiic/gallery/blob/main/package.json#L20 . Then the default sass installed with `@farmfe/plugin-sass` will be used.

I'll make plugin-sass compatible with latest sass-embedded later.

**Comment by wre232114** [comment]
issues found when trying to upgrade to latest sass-embedded. see: https://github.com/ahabhgk/sass-embedded-host-rust/issues/71
You can use default sass provided by `@farmfe/plugin-sass` for now. I'll upgrade it as soon as above issue fixed

**Comment by niuiic** [comment]
I removed sass-embedded, but no luck. Also tried installing `@farmfe/plugin-sass-linux-x64-gnu` but not work.

These two not found.

`@farmfe/plugin-sass/index.js`
```
if (existsSync(join(currentDir, './npm/linux-x64-gnu/index.farm'))) {
  52   │           binPath = join(currentDir, './npm/linux-x64-gnu/index.farm');
  53   │         } else {
  54   │           binPath = require.resolve('@farmfe/plugin-sass-linux-x64-gnu');
  55   │         }
```
css:build: cache miss, executing 0b4694a034d02853
css:build:
css:build: > css@0.0.0 build /home/niuiic/Documents/projects/gallery/apps/css
css:build: > farm build
css:build:
css:build: [ Farm ] Using config file at /home/niuiic/Documents/projects/gallery/apps/css/farm.config.ts
css:build: thread '<unnamed>' panicked at '
css:build:  sass-embedded init error: No such file or directory (os error 2),
css:build:  Please try to install manually. eg:
css:build:  pnpm install sass-embedded-linux-x64
css:build: ', rust-plugins/sass/src/lib.rs:140:9
css:build: note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
css:build: fatal runtime error: Rust cannot catch foreign exceptions
css:build:  ELIFECYCLE  Command failed.
css:build: ERROR: command finished with error: command (/home/niuiic/Documents/projects/gallery/apps/css) pnpm run build exited (-1)
```

**Comment by wre232114** [comment]
I'll try your project and fix this issue.

By the way, we are going to replace `sass-embedded` with `grass`, see this pr https://github.com/farm-fe/farm/pull/453.
In the future, sass-embedded is only used in JS Plugin

**Comment by wre232114** [comment]
fixed in `@farmfe/plugin-sass@0.2.6`

**Comment by niuiic** [comment]
Thanks
